eng/doc/fips/UserGuide.md

```diff
@@ -503,13 +681,31 @@ func sha1.New() hash.Hash
 
 New returns a new hash.Hash computing the SHA1 checksum.
 
-**Return values**
+**Implementation**
+
+<details><summary>OpenSSL</summary>
+
+The hash is generated using [EVP_MD_CTX_new] and [EVP_DigestInit_ex] with the algorithm [EVP_sha1].
 
 The hash.Hash methods are implemented as follows:
 
-- `Write(p []byte) (int, error)` using [SHA1_Update](https://www.openssl.org/docs/manmaster/man3/SHA1_Update.html).
-- `Sum(in []byte) []byte` using [SHA1_Final](https://www.openssl.org/docs/manmaster/man3/SHA1_Final.html).
-- `Reset()` using [SHA1_Init](https://www.openssl.org/docs/manmaster/man3/SHA1_Init.html).
+- `Write` using [EVP_DigestUpdate].
+- `Sum(` using [EVP_DigestFinal].

```

Comment:
Looks like `Sum(` should be `Sum`, a few more times in the doc too.

Reply:
Done!